client/settings/settings-manager/index.js

```diff
@@ -121,10 +112,14 @@ const SettingsManager = () => {
 					</LoadableSettingsSection>
 				</SettingsSection>
 			) }
-			<SettingsSection Description={ PaymentRequestDescription }>
+			<SettingsSection Description={ ExpressCheckoutDescription }>
 				<LoadableSettingsSection numLines={ 20 }>
 					<ErrorBoundary>
-						<PaymentRequest />
+						<ExpressCheckout
+							isPlatformCheckoutFeatureFlagEnabled={
+								isPlatformCheckoutFeatureFlagEnabled
+							}
```

Comment:
Rather than using prop drilling, you can use `useContext( WCPaySettingsContext )` within the `ExpressCheckout` component. That way its dependencies are more isolated.

Reply:
My initial thought was to declare feature flags all in one location, but I'm happy to go with this approach as well. Changed in https://github.com/Automattic/woocommerce-payments/pull/3738/commits/24890399a0ce14c72a9d8fae5fbc5a80ff690a03

Reply:
Looks good, thanks! :)